src/App.tsx

```diff
@@ -1,25 +1,88 @@
+import React, { useState } from 'react';
 import './App.scss';
+import classNames from 'classnames';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
 export const App = () => {
+  const [userValue, setUserValue] = useState(0);
+  const [todoTitle, setTodoTitle] = useState('');
+  const [titleError, setTitleError] = useState(false);
+  const [userError, setUserError] = useState(false);
+
+  const idCreator = () => todosFromServer.length + 1;
```

Comment:
The way you create new ids for todos could potentially cause issues as it doesn't guarantee unique ids if a todo is removed. Consider using some library to generate unique ids, like uuid.